diagnostic_rt_qpcr/operation_types/run_qpcr/protocol.rb

```diff
@@ -134,7 +133,7 @@ def start_thermocyclers(paired_ops)
   end
 
   def associate_measurement(file_name:, plate:)
-    file = upload_data(file_name, 1, 4)
+    file = uploadData(file_name, 1, 4)
```

Comment:
Should be snake_case, but I understand that this is coming from another library so don't change.